docs/docs/1.1.1/deployments/how-to-multi-zone.md

```diff
@@ -0,0 +1,311 @@
+
+### Apply TrafficPermission policy
+
+You must also apply a [TrafficPermission policy](../policies/traffic-permissions.md). Traffic permissions also require enabling mTLS. Kuma creates a default `TrafficPermission` policy that allows all the communication between all the services when a new `Mesh` is created. You should take care to create appropriate policies to restrict traffic between services. 
+
+### Ingress requirements
+
+Cross-zone communication between services is available only if Ingress has a public address and public port.
+
+On Kubernetes, Kuma automatically tries to pick up the public address and port. Depending your load balancing implementation, you might need to wait a few minutes for Kuma to get the address. 
+
+### Cross-communication details
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+
+To view the list of service names available for cross-zone communication, run:
+
+```bash
+$ kubectl get dataplanes -n echo-example -o yaml | grep service
+           service: echo-server_echo-example_svc_1010
+```
+
+To consume the example service only within the same Kuma zone, you can run:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server:1010
+```
+
+To consume the example service across all zones in your Kuma deployment (that is, from endpoints ultimately connecting to the same global control plane), you can run either of:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh:80
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh:80
+```
+
+And if your HTTP clients take the standard default port 80, you can the port value and run either of:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh
+```
+
+Because Kuma on Kubernetes relies on transparent proxy, `kuma-dp` listens on port 80 for all virtual IPs that are assigned to services in the `.mesh` DNS zone. The DNS names are rendered RFC compatible by replacing underscores with dots.
+
+:::
+::: tab "Universal"
+
+With a hybrid deployment, running in both Kubernetes and Universal mode, the service tag should be the same in both environments (e.g `echo-server_echo-example_svc_1010`):
+
+```yaml
+type: Dataplane
+mesh: default
+name: backend-02 
+networking:
+  address: 127.0.0.1
+  inbound:
+  - port: 2010
+    servicePort: 1010
+    tags:
+      kuma.io/service: echo-server_echo-example_svc_1010
+```
+
+With a Universal control plane, the service tag has no such limitation.
+
+REVIEWER: DOES THIS ^ APPLY TO A HYBRID ENVIRONMENT WHERE THE CONTROL PLANE RUNS IN UNIVERSAL MODE? DOES THIS REFER TO THE GLOBAL CONTROL PLANE? CAN WE CLARIFY THE DISTINCTION BETWEEN THESE TWO CASES (THAT IS, THE HYBRID AND THIS SECOND CASE WHICH MIGHT OR MIGHT NOT BE RELEVANT TO THE FIRST ONE)?
+
+To consume a distributed service in a Universal deployment, where the application address is `http://localhost:20012`:
```

Comment:
This is the "old" still supported mode to consumer services on Universal. We should put forward the Transparent Proxy mode here. Please take the content from here:
https://kuma.io/docs/1.1.1/networking/transparent-proxying/